webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const HTMLWebpackPlugin = require('html-webpack-plugin');
+
```

Comment:
Webpack should have separate configs for dev and prod https://webpack.js.org/guides/production/

Reply:
Dev https://webpack.js.org/guides/development/

Reply:
Prod https://webpack.js.org/guides/production/